src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/model/ResultOrErrors.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.model;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+
+/**
+ * Result of retrieving/parsing an object.
+ * <p>
+ * Contains either the retrieved object or a list of error messages.
+ * </p>
+ */
+public class ResultOrErrors<T> {
```

Comment:
could `TransformationResult` from the library be used instead?

Reply:
ok, nevermind this is not even close to actual transformation, only used for parsing json.